echemdb/make_pages.py

```diff
@@ -106,23 +100,17 @@ def create_element_pages(elementname):
     templatemd['data']['element'] = elementname
     templatemd['title'] = 'echemdb - {} CV data'.format(elementname)
 
-    target = copy.deepcopy(TARGET_FOLDERS['elements']).replace('tobesubstituted', elementname)
```

Comment:
There is no need to `deepcopy` a string. A `replace` does not mutate strings but just returns a modified copy.